src/Evolution/Systems/NewtonianEuler/Tags.hpp

```diff
@@ -85,6 +79,26 @@ struct SoundSpeedSquared : db::SimpleTag {
   using type = Scalar<DataType>;
 };
 
+/// The characteristic speeds.
+template <size_t Dim>
+struct CharacteristicSpeeds : db::SimpleTag {
+  using type = std::array<DataVector, Dim + 2>;
+};
+
+// @{
+/// The characteristic fields of the NewtonianEuler system.
+struct UMinus : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+template <size_t Dim>
+struct U0 : db::SimpleTag {
```

Comment:
I put these names down because it was easy, but I don't love them either. I will be very happy to make a change if you or anyone else can suggest better names! :)

Reply:
If the `U0` corresponds to the momentum/velocity then maybe `VMomentum`?